certora/steward/specs/rules.spec

```diff
@@ -313,7 +313,7 @@ rule updateGsmExposureCap__correctness() {
     assert EXPOSURE_CAP==newExposureCap;
     
     uint128 exposure_cap_after = EXPOSURE_CAP;
-    assert exposure_cap_before <= exposure_cap_after &&
+    assert 0 <= to_mathint(exposure_cap_after) &&
```

Comment:
This is a correct conversion.
Note that the first clause is trivially correct since `exposure_cap_after` is a uint128 and the solidity will make sure to revert if the update function cause an underflow.
That's still fine to leave in.

Reply:
fixed in [cdb664e](https://github.com/aave/gho-core/pull/406/commits/cdb664ef243baad64786b6d89fb0a1dc21e850a5)